app.js

```diff
@@ -3,16 +3,26 @@ const app = express();
 app.use (express.json()) ;
 const {
     getTopics,
-    addToApi
+    getData
 } = require('./controllers/controller.js');
 
 
 app.get('/api/topics', getTopics);
-app.get('/api', addToApi);
+app.get('/api', getData);
+
+app.all('/*', (req, res, next) => {
+  res.status(404).send({msg: "Invalid api endpoint"});
+});
```

Comment:
Nice! However, I can't seem to see a test for this 404. Could you add a test trying to access an endpoint that does not exist and checking that the status code and message match what you have in this line?